src/client/Client.js

```diff
@@ -108,24 +109,49 @@ export default class Client {
     }
 
     /**
+     * @deprecated
      * @param {NetworkName} networkName
      * @returns {this}
      */
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     setNetworkName(networkName) {
-        this._network.setNetworkName(networkName);
-        return this;
+        //uses custom NetworkName type
```

Comment:
Not going to block this PR for this, but in the future can we use `//<space><comment>` instead of `//<comment>` for consistency. That or use JSDoc comments.